### PR TITLE
Min max for input data

### PR DESCRIPTION
**Pull request by marjanfamili**
This PR addresses #143 

An optional variable can be called in the input file which shifts the min of the data to 0 and max to 1

### CODE REVIEW

File: avae/data.py [hunk 1]
```diff
@@ -188,6 +194,9 @@ def __init__(
         amatrix=None,
         classes=None,
         transform=None,
+        gaussian_blur=False,
+        normalise=False,
+        shift_min=False,
```
Comment (jolaem) [inline avae/data.py:199]: Add to shift-mean to docstring.

File: avae/evaluate.py [hunk 2]
```diff
@@ -10,15 +10,30 @@
...
+    state,
+    meta,
+    lim,
+    splt,
+    batch_s,
+    collect_meta,
+    use_gpu,
+    gaussian_blur,
+    normalise,
+    shift_min,
```
Comment (jolaem) [inline avae/evaluate.py:24]: Add to shift-mean to docstring.

File: avae/train.py [hunk 3]
```diff
@@ -44,6 +48,9 @@ def train(
     recon_fn,
     use_gpu,
     model,
+    gaussian_blur,
+    normalise,
+    shift_min,
```
Comment (jolaem) [inline avae/train.py:53]: Add to shift-mean to docstring.

File: avae/train.py [hunk 4]
```diff
@@ -122,6 +134,9 @@ def train(
         eval=False,
         affinity=affinity,
         classes=classes,
+        gaussian_blur=gaussian_blur,
+        normalise=normalise,
+        shift_min=shift_min,
```
Comment (jolaem) [inline avae/train.py:139]: Add to shift-mean to docstring.

File: configs/avae-test-config.yml [hunk 5]
```diff
@@ -3,6 +3,8 @@
 datapath: data/shrec21_full_dataset/subtomo_files
 affinity: data/shrec21_full_dataset/subtomo_files/affinity.csv
 classes: data/shrec21_full_dataset/subtomo_files/classes.csv
+state : /your_path_to_saved_state/state.pt
+restart: True
 limit: 1000
 split: 20
```
Comment (jolaem) [inline configs/avae-test-config.yml:9]: Add gaussian_blur, normalise and shift_mean to default config

File: avae/data.py [hunk 6]
```diff
@@ -23,6 +23,9 @@ def load_data(
     eval: bool = True,
     affinity=None,
     classes=None,
+    gaussian_blur=False,
+    normalise=False,
+    shift_min=False,
```
[the remark this answers is not in the document]
Comment (crangelsmith) [reply avae/data.py:28]: fixed in [f1c9aca](https://github.com/alan-turing-institute/affinity-vae/pull/144/commits/f1c9acad22d140675956401d316dbd196aea0663)

File: avae/data.py (hunk 1 above)
Comment (crangelsmith) [reply avae/data.py:216]: fixed in [f1c9aca](https://github.com/alan-turing-institute/affinity-vae/pull/144/commits/f1c9acad22d140675956401d316dbd196aea0663)

File: avae/evaluate.py (hunk 2 above)
Comment (crangelsmith) [reply avae/evaluate.py:24]: fixed in [9d23df5](https://github.com/alan-turing-institute/affinity-vae/pull/144/commits/9d23df5a61fa6f06f3200226aeeb560744fc04b9)

File: avae/train.py (hunk 4 above)
Comment (crangelsmith) [reply avae/train.py:145]: fixed in [5876b00](https://github.com/alan-turing-institute/affinity-vae/pull/144/commits/5876b00ca5e0015fecbeed2d7376a3eb90f58af9)

File: avae/train.py (hunk 3 above)
Comment (crangelsmith) [reply avae/train.py:53]: fixed in [5876b00](https://github.com/alan-turing-institute/affinity-vae/pull/144/commits/5876b00ca5e0015fecbeed2d7376a3eb90f58af9)

File: configs/avae-test-config.yml (hunk 5 above)
Comment (crangelsmith) [reply configs/avae-test-config.yml:9]: fixed in [c061edf](https://github.com/alan-turing-institute/affinity-vae/pull/144/commits/c061edf05f2e7f9114d7cc1a7100b2ec826dd691)

**Pull request merged**